File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/wishList/Slist.java

package com.artmurka.artmurkaapp.data.model.pojo.itemlist.wishList;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Slist {

    @SerializedName("name")
    @Expose
    private String name;
    @SerializedName("field")
    @Expose
    private String field;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getField() {
        return field;
    }

    public void setField(String field) {
        this.field = field;
    }

}

File: app/src/main/java/com/artmurka/artmurkaapp/data/model/pojo/itemlist/wishList/Slist.kt
package com.artmurka.artmurkaapp.data.model.pojo.itemlist.wishList

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Slist {

    @SerializedName("name")
    @Expose
    var name: String? = null
    @SerializedName("field")
    @Expose
    var field: String? = null

}
